x11-libs/wxgtk/wxgtk-3.1.5.recipe

```diff
@@ -0,0 +1,198 @@
+SUMMARY="Cross-platform GUI library"
+DESCRIPTION="wxWidgets is a C++ library that lets developers create \
+applications for Windows, macOS, Linux and other platforms with a single \
+code base. It has popular language bindings for Python, Perl, Ruby and many \
+other languages, and unlike other cross-platform toolkits, wxWidgets gives \
+applications a truly native look and feel because it uses the platform's \
+native API rather than emulating the GUI. It's also extensive, free, \
+open-source and mature."
+HOMEPAGE="https://www.wxwidgets.org/"
+COPYRIGHT="1998-2019 Julian Smart, Robert Roebling et al"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/wxWidgets/wxWidgets/releases/download/v$portVersion/wxWidgets-$portVersion.tar.bz2"
+CHECKSUM_SHA256="d7b3666de33aa5c10ea41bb9405c40326e1aeb74ee725bb88f90f1d50270a224"
+SOURCE_DIR="wxWidgets-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libSoVersion="3.0.0"
```

Comment:
libVersion (on a check on 32bit) is 5.0.0, I see no libSoVersion here (hence the debug package fails at the moment)

Reply:
I've update it please check again.